bin-src/lib/getOptions.js

```diff
@@ -50,7 +50,7 @@ export default async function getOptions({ argv, env, flags, log, packageJson })
     originalArgv: argv,
 
     buildScriptName: flags.buildScriptName,
-    outputDir: flags.outputDir,
+    outputDir: Array.isArray(flags.outputDir) ? takeLast(flags.outputDir) : undefined,
```

Comment:
Doesn't `takeLast` already include this ternary?

Reply:
The value should always be an array of strings (thanks to `isMultiple: true`), but if it isn't -somehow- I want to return `undefined`.

Reply:
<img width="649" alt="image" src="https://user-images.githubusercontent.com/132554/141856604-ef16d2b3-8028-486f-b0fc-9429e5f9b886.png">

Are you concerned about the possibility of the flag not being an array but also not being `undefined`? Is that actually possible with the library? Would that concern not apply to other options we are `takeLast`-ing? (In which case should `takeLast` just return `undefined` in the non-array case?)